contracts/utils/addresses.js

```diff
@@ -184,6 +184,9 @@ addresses.mainnet.CurveOETHGauge = "0xd03BE91b1932715709e18021734fcB91BB431715";
 addresses.mainnet.CVXETHRewardsPool =
   "0x24b65DC1cf053A8D96872c323d29e86ec43eB33A";
 
+addresses.mainnet.CompoundingStakingStrategyProxy =
+  "0x840081c97256d553A8F234D469D797B9535a3B49";
```

Comment:
This address has nothing deployed on the mainnet yet: 
https://etherscan.io/address/0x840081c97256d553A8F234D469D797B9535a3B49

Reply:
Good pickup. The proxy had been deployed to 0xaF04828Ed923216c77dC22a2fc8E077FDaDAA87d but the contract name is `OETHCurveAMOProxy` and not `CompoundingStakingSSVStrategyProxy`.
I raised a ticket with Etherscan a few weeks ago to rename. I'll see if I can find what happened to the request. Otherwise it might be easier to deploy a new one.

Reply:
I've changed this address to `addresses.unitTests.CompoundingStakingStrategyProxy` as the unit testing proofs have been generated with the staking strategy at 0x840081c97256d553A8F234D469D797B9535a3B49

See commit https://github.com/OriginProtocol/origin-dollar/pull/2559/commits/512f6f7b6b04f2194272b09bde8a04432df589c7

Reply:
nice thanks